vcpkg/maintainers/publishing-to-a-git-registry.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Publishing to a Git registry
+description: Learn to publish vcpkg ports to a Git-based registry
+author: vicroms
+ms.author: viromer
+ms.date: 7/21/2023
+ms.prod: vcpkg
+---
+# Publishing to a Git registry
+
```

Comment:
Would it be useful to provide some sort of introduction before the steps? I think it could help readers decide why and when a user might want to publish vcpkg ports to a git registry. Maybe even define what git-based registry?